GLTF/GLTFAsset.h

```diff
@@ -56,10 +57,20 @@ namespace GLTF
 
     typedef std::map<std::string, std::shared_ptr<GLTFOutputStream> > NameToOutputStream;
 
+	// Conceptually it would be nice to reuse a lambda,
+	// but then we would have to pass it in everywhere
+	struct UniqueIDHasher {
+        size_t operator()(const COLLADAFW::UniqueId& uniqueID) {
```

Comment:
This needs to be size_t operator()(const COLLADAFW::UniqueId& uniqueID) **const** to build on Windows